src/components/SignIn.tsx

```diff
@@ -340,10 +360,11 @@ const SignIn = ({ onWeb3ProviderSet, onWeb3AuthInstanceSet }: SignInProps) => {
         { title: 'Twitch', icon: <img src={iconTwitch} alt="twitch" />, onClick: () => loginWithOpenLogin('twitch') },
       ],
       web3: [
-        { title: 'MetaMask', icon: <img src={iconMetamask} alt="metamask" />, onClick: () => loginWithAdapter(WALLET_ADAPTERS.METAMASK) },
-        { title: 'WalletConnect', icon: <img src={iconWalletConnect} alt="walletconnect" />, onClick: () => loginWithAdapter(WALLET_ADAPTERS.WALLET_CONNECT_V1) },
-        { title: 'Torus EVM', icon: <img src={iconTorus} alt="torus" />, onClick: () => loginWithAdapter(WALLET_ADAPTERS.TORUS_EVM) },
-        { title: 'Coinbase', icon: <img src={iconCoinbase} alt="coinbase" />, onClick: () => loginWithAdapter(WALLET_ADAPTERS.COINBASE) },
+        ...connectors.map((connector) => ({
+          title: connector.name,
+          icon: iconByName[connector.name],
```

Comment:
connectors have ID, names are super error prone due case sensitivity or branding changes etc, please use connector ID for such

Reply:
agreed though the id's are like `metaMask` , `coinbaseWallet` etc